setup.py

```diff
@@ -41,9 +41,9 @@ def create_extension() -> setuptools.Extension:
 
     extra_compile_args = {
         "windows": ["/DANTLR4CPP_STATIC", "/Zc:__cplusplus"],
```

Comment:
You also need to add a C++17 flag here (spoiler: it's a different one) but I would also have to google it.

Reply:
```suggestion
        "windows": ["/DANTLR4CPP_STATIC", "/Zc:__cplusplus", "/std:c++17"],
```